src/components/common/TmSessionSignIn.vue

```diff
@@ -85,6 +100,9 @@ export default {
       }))
     }
   },
+  created() {
+    this.$store.dispatch("loadAccounts")
```

Comment:
Why do we need this now?

Reply:
This fixes the other bug, that accounts were not loaded after refreshing.

It has nothing to do with the universal sign in. It is just an additional thing, fixing a bug (yes, it should have been done in another PR but it is a really easy fix, 3 lines)

Reply:
I think this is solving the issue at the wrong position. But let's keep in the back of our head.

Reply:
It is only fixing what I told you xD 

What is this wrong position bug?